compute/instances/reference-content/cost-optimized.mdx

```diff
@@ -49,6 +49,22 @@ See below the technical specifications of PLAY2 and PRO2 Instances:
 | Sizing   	                      | From 1 to 32 vCPUs <br />From 2 to 128 GB RAM                                 |
 | vCPU:RAM ratio                  | Various<br />(1:2, 1:4)                                                       |
 
+## COP-ARM Instances
+
+An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the world of development and general purpose computing. Their ARM-based architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of ARM architecture.
+
+See below the technical specifications of COP-ARM Instances:
+
+| Range                         	| Cost-Optimized                                                              	|
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | COPARM1                                                             	        |
+| Availability Zone   	          | PAR2                                                                          |
+| Storage   	                    | Block                                                                         |
+| Max. Bandwidth   	              | From 100 Mbps to 3.2 Gbps                                                     |
```

Comment:
Shouldn't it be _From 200 Mbps_ ?

Reply:
I saw 100 Mbps - but maybe my source was outdated. I'll update it.

Reply:
```suggestion
| Max. Bandwidth   	              | From 200 Mbps to 3.2 Gbps                                                     |
```